.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Fixes #replace_this_text_with_the_issue_number
+
+### What changes did you make and why did you make them ?
+
+  -
+  -
+  -
+
+### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
+<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
+<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->
+
+<details>
```

Comment:
Did you want to put this in a code block? 
like 

```
>```
<details>
<summary>Visuals after changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

</details>
>```
```

Reply:
most md readers parse html, so this gets parsed with a missing image.. might need to put it in a code block.

Reply:
Fixes #replace_this_text_with_the_issue_number

### What changes did you make and why did you make them ?

  -
  -
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details><summary>Visuals before changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

</details>

<details>
<summary>Visuals after changes are applied</summary>

![image](Paste_Your_Image_Link_Here_After_Attaching_Files)

</details>

Reply:
This is the default template that the hackforla/website team uses. The idea is you replace the line:
```
![image](Paste_Your_Image_Link_Here_After_Attaching_Files)
```

with your screenshot images. It helps save time so you don't have to type out <details><summary><summary/></details> in order to prevent an image from taking up too much space

Reply:
Right! I think that is awesome.. My bad that I did not explain myself more clearly, I was just thinking that because the html is parsed by the markdown viewer on github, it might be hard to copy and paste. so it might not be a bad idea to wrap and copy-paste-able stuff in a markdown code block.

I am going to go ahead and merge. that was just a thought

Reply:
@jbubar 

the markdown template is automatically inserted into the PR creation comment field because the markdown template is named properly and located in the `.github/` folder


 There is no copy and pasting necessary.

Unless I am still misunderstanding you?